nuget/lib/dependabot/nuget/native_helpers.rb

```diff
@@ -1,6 +1,8 @@
 # typed: true
 # frozen_string_literal: true
 
+require_relative "nuget_config_credential_helpers.rb"
```

Comment:
The `.rb` extension isn't necessary for `require_relative`. I don't think it causes any harm but since we don't do it elsewhere, might as well leave it off

```suggestion
require_relative "nuget_config_credential_helpers"
```